src/ui_ng/lib/src/create-edit-endpoint/create-edit-endpoint.component.ts

```diff
@@ -91,7 +92,8 @@ export class CreateEditEndpointComponent implements AfterViewChecked, OnDestroy
                 (this.target.endpoint && this.target.endpoint.trim() !== "") ||
                 (this.target.name && this.target.name.trim() !== "") ||
                 (this.target.username && this.target.username.trim() !== "") ||
-                (this.target.password && this.target.password.trim() !== ""));
+                (this.target.password && this.target.password.trim() !== "")) ||
+                (this.target.insecure !== this.initVal.insecure);
```

Comment:
Here is for create new, that means no old values, all are new values. No need to check if == with the original value, right?